### PR TITLE
Add base method to get power status of Azure VM

### PR DESCRIPTION
**Pull request by suchita-g**
Signed-off-by: Suchita Gatfane <sgatfane@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/azure_utils.py [hunk 1]
```diff
@@ -220,6 +220,24 @@ def get_vm_instance(self, vm_name):
         )
         return vm
 
+    def get_vm_power_status(self, vm_name):
+        """
+        Get the running status of VM
+        Args:
```
Comment (sidhant-agrawal) [inline ocs_ci/utility/azure_utils.py:226]: Add empty line

File: ocs_ci/utility/azure_utils.py [hunk 2]
```diff
@@ -220,6 +220,24 @@ def get_vm_instance(self, vm_name):
...
+           vm_name (str): Azure VM name
+        Returns :
```
Comment (sidhant-agrawal) [inline ocs_ci/utility/azure_utils.py:228]: Add empty line here as well

File: ocs_ci/utility/azure_utils.py [hunk 3]
```diff
@@ -220,6 +220,24 @@ def get_vm_instance(self, vm_name):
...
+           str: Power status of Azure VM
+
+        """
+        vm = self.compute_client.virtual_machines.get(
+            self.cluster_resource_group, vm_name, expand="instanceView"
+        )
+        vm_statuses = vm.instance_view.statuses
+        vm_power_state = (
+            vm_statuses.__len__() >= 2 and vm_statuses[1].code.split("/")[1]
```
Comment (mbukatov) [inline ocs_ci/utility/azure_utils.py:237]: Use `len(vm_statuses)` instead.

Comment (suchita-g) [reply ocs_ci/utility/azure_utils.py]: Modified change.

**Review by sidhant-agrawal — DISMISSED** [review]

**Review by jilju — APPROVED** [review]

File: ocs_ci/utility/azure_utils.py [hunk 4]
```diff
@@ -220,6 +220,24 @@ def get_vm_instance(self, vm_name):
         )
         return vm
 
+    def get_vm_power_status(self, vm_name):
+        """
+        Get the running status of VM
```
Comment (ebenahar) [inline ocs_ci/utility/azure_utils.py:225]: shouldn't it be `Get the power status of VM`?

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/utility/azure_utils.py (hunk 4 above)
Comment (suchita-g) [reply ocs_ci/utility/azure_utils.py]: yes. Updated

**Review by ebenahar — APPROVED** [review]

**Review by mbukatov — APPROVED** [review]

**Comment by suchita-g** [comment]
Verified test result: http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/sgatfane-2701/sgatfane-2701_20210127T043340/logs/test_report_get_test_power_state.html

**Pull request merged**